include/mlir/Analysis/Dominance.h

```diff
@@ -90,6 +90,12 @@ class DominanceInfo : public detail::DominanceInfoBase</*IsPostDom=*/false> {
   bool properlyDominates(Block *a, Block *b) {
     return super::properlyDominates(a, b);
   }
+
+  /// Return the dominanceInfos from the Region containing block A.
+  llvm::DomTreeNodeBase<Block> *getNode(Block *a);
+
+  /// Update the DFS numbers on all the dominanceInfos.
```

Comment:
dominanceInfos doesn't make sense here either, can we just say 'Updated the internal DFS numbers for the dominance nodes'

Reply:
change pushed